docs/content/misc/downloading.ngdoc

```diff
@@ -29,58 +29,94 @@ example points to the minified version 1.5.6:
-<http://docs.angularjs.org/> documentation app. These files are handy to see the older version of
-our docs, or even more importantly, view the docs offline.
-
-* __`i18n`__ - this directory contains locale specific `ngLocale` angular modules to override the defaults
-defined in the `ng` module.
+This option is for those who want to work with Angular offline, or those who want to host the
+Angular files on their own servers.
+
+If you navigate to https://code.angularjs.org/, you'll see a directory listing with all of the
+Angular versions since we started releasing versioned build artifacts. Each directory contains all
+artifacts that we released for a particular version. Download the version you want and have fun.
+
+<div class="alert alert-warning">
+  You can ignore directories starting with `2.` (e.g. `2.0.0-beta.17`) — they are not related to
+  AngularJS. They contain build artifacts from [Angular 2](https://angular.io) versions.
+</div>
+
+<br />
+Each directory under https://code.angularjs.org/ includes a set of files that comprise the
+corresponding version. All JavaScript files (except for `angular-mocks` which is only used during
+development) come in two flavors — one suitable for development, the other for production:
+
+* __`<filename>.js`__ — These files are non-obfuscated, non-minified, and human-readable by opening
+  them in any editor or browser. In order to get better error messages during development, you
+  should always use these non-minified scripts.
+
+* __`<filename>.min.js`__ — These are minified and obfuscated versions, created with the
+  [Closure compiler](https://developers.google.com/closure/compiler/). Use these versions for
+  production in order to minimize the size of the application that is downloaded by your user's
+  browser.
+
+* __`<filename>.min.js.map`__ — These are
+  [sourcemap files](http://www.html5rocks.com/en/tutorials/developertools/sourcemaps/). You can
+  serve them alongside the `.min.js` files in order to be able to debug the minified code (e.g. on a
+  production deployment) more easily, but without impacting performance.
+
+<br />
+The set of files included in each version directory are:
+
+* __`angular.zip`__ — This is a zip archive that contains all of the files released for this Angular
+  version. Use this file to get everything in a single download.
+
+* __`angular.js`__ — The core Angular framework. This is all you need to get your Angular app
+  running.
+
+* __`angular-csp.css`__ — You only need this file if you are using
+  [CSP (Content Security Policy)](https://developer.mozilla.org/en/Security/CSP). See
+  {@link directive:ngCsp here} for more info.
+
+* __`angular-mocks.js`__ — This file contains an implementation of mocks that makes testing angular
+  apps even easier. Your unit/integration test harness should load this file after `angular.js` is
+  loaded.
+
+* __`angular-loader.js`__ — Module loader for Angular modules. If you are loading multiple 
```

Comment:
I left it as it was previously (probably outdated as the rest of the content :smiley:).

TBH, I've never used the loader. So, does it mean you need to load the loader (synchronously), then angular.js (synchronously), then other modules (asynchronously and in any order) ?